### PR TITLE
LPD-34521 Review tests in client-extension-web

### PR DESCRIPTION
**Pull request by thektan**
https://liferay.atlassian.net/browse/LPD-34521

Fixes frontend-infra-owned tests in `client-extension-web`.

- Slightly unsure how acceptable this change is f694a3a5c73a3899a3242c85ce6823a0fd6e98a8, but it seemed like `layout.friendlyUrlPath` wasn't defined when the layout was created with [`apiHelpers.jsonWebServicesLayout.addLayout`](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/fixtures/isolatedLayoutTest.ts#L69-L76).
  - For example, `layout.friendlyUrlPath` is created when it's created with a site using [`apiHelpers.headlessDelivery.createSitePage`](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-web/clayTable.spec.ts#L37-L47).
  - I also found places where the URL path was being created with `layout.friendlyURL` instead of `layout.friendlyUrlPath` as well https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts#L97

### CODE REVIEW

**Comment by thektan** [comment]
ci:test:playwright

**Comment by thektan** [comment]
None of the failures seem related, but I did notice 2 additional sample hashes that needed be updated so I made that change as well.

**Comment by thektan** [comment]
ci:test:playright

**Comment by thektan** [comment]
ci:test:playwright

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class PageEditorPage {
 		await this.page.goto('/');
 
 		await this.page.goto(
-			`/web${siteUrl || '/guest'}${layout.friendlyUrlPath}?p_l_mode=edit${doAsUserId ? '&doAsUserId=' + doAsUserId : ''}`
+			`/web${siteUrl || '/guest'}${layout.friendlyUrlPath || layout.friendlyURL}?p_l_mode=edit${doAsUserId ? '&doAsUserId=' + doAsUserId : ''}`
```
Comment (markocikos) [inline modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:67]: Inconsistency in what is used in `layout` is not good, but improving that is outside of scope of this task. This quick&dirty-ish fallback solution is fine by me :+1:

**Review by markocikos — APPROVED** [review]
@thektan LGTM! Just a comment on inconsistency inline.

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts (hunk 1 above)
Comment (dsanz) [reply modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:67]: I don't have objections either

**Comment by thektan** [comment]
After looking at the test results, all `client-extension-web` tests are passing and none of the failures seem related.

**Comment by thektan** [comment]
ci:forward

**Pull request closed without merging**